code/modules/swarmers/swarmer.dm

```diff
@@ -465,7 +460,11 @@
 	. = ..()
 	if(!.)
 		return
-	if(!istype(target, /mob/living/simple_animal) || !istype(firer, /mob/living/simple_animal/hostile/swarmer))
+	if((!istype(target, /mob/living/simple_animal) && !ishuman(target)) || !istype(firer, /mob/living/simple_animal/hostile/swarmer))
```

Comment:
do you really need if(it's not a simple animal, and it's not a human) as a check? what does this catch?

Reply:
The beam teleporting cyborgs and other carbons.

Reply:
ahhh I see